cli/exec.go

```diff
@@ -73,7 +75,7 @@ func (e *ExecCommand) runExec(args []string) error {
 		AttachStderr: !e.Detach,
 		AttachStdout: !e.Detach,
 		AttachStdin:  !e.Detach && e.Interactive,
-		Privileged:   false,
+		Privileged:   e.Privileged,
```

Comment:
there is only client side change. And I didn't see that the daemon can use the field. 

I think you should add the `NoNewPrivileges` here.
https://github.com/alibaba/pouch/blob/8abf067257260a1570ed0a09bff2b16df9a1b899/daemon/mgr/container_exec.go#L88-L98

Reply:
two different things, `NoNewPrivileges`  https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt , `Privileged` here related with container level

Reply:
anyway, LGTM. and wait for the CI.